examples/tv-casting-app/tv-casting-common/src/CastingServer.cpp

```diff
@@ -39,7 +44,6 @@ void CastingServer::InitServer()
     {
         return;
     }
-    DeviceLayer::PersistedStorage::KeyValueStoreMgrImpl().Init(CHIP_CONFIG_KVS_PATH);
```

Comment:
Doesn't the android app also need to initialize persistent storage?

Reply:
The android app already initialized the KeyValueStoreMgrImpl and in the latest diff here, we have the linux app doing the same on its own.